packages/react-app/src/contexts/BridgeContext.jsx

```diff
@@ -148,11 +154,35 @@ export const BridgeProvider = ({ children }) => {
     [setToken, fromToken, toToken],
   );
 
+  const checkForCustomToken = useCallback(async () => {
+    const { token } = isTokenInjected;
+    if (!fromToken && !toToken) {
+      const tokenList = await fetchTokenList(providerChainId);
+      const customToken = tokenList.find(t => t.symbol.toLowerCase() === token);
```

Comment:
I see that you're  using the token query param as the symbol. But there could be multiple tokens with the same symbol.
Let the query param be the tokenAddress instead.

Reply:
Noted.

Reply:
As discussed over, since we would be supporting BridgeDropdown #150, the query param for the chainId is gonna stay as `from`